public/web/WebView.h

```diff
@@ -375,8 +375,10 @@ class WebView : public WebWidget {
 
     // Accessibility -------------------------------------------------------
 
+#ifndef DISABLE_ACCESSIBILITY
     // Returns the accessibility object for this view.
     virtual WebAXObject accessibilityObject() = 0;
```

Comment:
Or to make modification easier,  keep .h file, but delete function body in .cpp file, the the linker will make sure no calls from content side.